recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
-{% set version = "0.6" %}
-{% set sha256 = "edcf3e908f0d873e569af593166416affa6e87e08bedb8bddca5f50fc9799f2f" %}
-{% set number = "1" %}
+{% set version = "0.7.1" %}
+{% set sha256 = "c7744df945fa90791ad9b4183a6a7dc8220d63a7b8a5f8f93ba62086f1e69e83" %}
+{% set number = "0" %}
 
 package:
   name: python-graphviz
   version: {{ version }}
 
 source:
-  fn: python-graphviz-{{ version }}.tar.gz
-  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
+  fn: graphviz-{{ version }}.zip
+  url: https://files.pythonhosted.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
   sha256: {{ sha256 }}
   patches:
     # needed due to the Windows batch script wrappers in conda-forge Graphviz
     # see https://github.com/conda-forge/graphviz-feedstock/blob/bbfe3c7be2a448dd11db165a18c3edfd5ee6a95d/recipe/bld.bat#L27-L32
-    - windows-bat.patch
+    - windows-bat.patch  # [win]
```

Comment:
Ok, that's fine. Didn't know we had issues with this as we already had the patch applied on all platforms before.